source/routing/raptor_api.cpp

```diff
@@ -1363,6 +1370,18 @@ void make_response(navitia::PbCreator& pb_creator,
         return;
     }
 
+    for (const auto stop_point_iter : *departures) {
```

Comment:
```suggestion
    for (const auto& stop_point_iter : *departures) {
```

Reply:
FYI : Always try to use `const auto&` :cat: to protect fastness and security

Reply:
051ab1a4d1bf54e95d2a5d195910e797830ad317